src/helpers/statics.ts

```diff
@@ -223,3 +223,57 @@ export const SOCIAL_URLS = [
     'weibo\\.com',
     'im\\.qq\\.com',
 ];
+
+// https://ads.tiktok.com/help/article/standard-events-parameters?lang=en
+export const TK_STANDARD_EVENTS = [
+  { eventDescription: "When a visitor adds their payment information at checkout.", eventName: "AddPaymentInfo" },
+  { eventDescription: "When a visitor adds an item to the cart.", eventName: "AddToCart" },
+  { eventDescription: "When a visitor adds an item to the wishlist.", eventName: "AddToWishlist" },
+  { eventDescription: "When an application submitted by a user is approved (e.g. loan, job, enrollment).", eventName: "ApplicationApproval" },
+  { eventDescription: "When a visitor signs up / completes registration.", eventName: "CompleteRegistration" },
+  { eventDescription: "When a visitor contacts you.", eventName: "Contact" },
+  { eventDescription: "When a visitor customizes a product.", eventName: "CustomizeProduct" },
+  { eventDescription: "When a visitor downloads something from your website.", eventName: "Download" },
+  { eventDescription: "When a visitor looks for location.", eventName: "FindLocation" },
+  { eventDescription: "When a visitor proceeds to checkout.", eventName: "InitiateCheckout" },
+  { eventDescription: "When a visitor makes a purchase.", eventName: "Purchase" },
+  { eventDescription: "When a visitor schedules an appointment.", eventName: "Schedule" },
+  { eventDescription: "When a visitor searches for something.", eventName: "Search" },
+  { eventDescription: "When a customer begins a free trial.", eventName: "StartTrial" },
+  { eventDescription: "When someone submits an application (e.g. for credit, job, admission).", eventName: "SubmitApplication" },
+  { eventDescription: "When a visitor submits a form.", eventName: "SubmitForm" },
+  { eventDescription: "When a visitor subscribes (e.g. to content or paid subscription).", eventName: "Subscribe" },
+  { eventDescription: "When a visitor views a specific content page.", eventName: "ViewContent" },
+  { eventDescription: 'This is the default pixel tracking page visits.', eventName: 'Pageview'}
+];
+
+//https://business-api.tiktok.com/portal/docs?rid=5ipocbxyw8v&id=1739585700402178
+export const TK_ADVANCED_MATCHING_PARAMETERS = {
```

Comment:
I think it's a good idea to avoid abbreviations for things like this, because it's not obvious that `TK` means TikTok.  Can you rename these constants to `TIKTOK_[...]` instead?